core/modules/user/user.install

```diff
@@ -1344,6 +1352,999 @@ function user_update_1023() {
   }
 }
 
+/**
+ * Add a "changed" column to the "users" table.
+ */
+function user_update_1024() {
+  // On older databases originating from Drupal 5, the "changed" column was
+  // renamed to "access" in system_update_136(), and the "changed" index on the
+  // "access" column may still persist. That column index needs to be removed
+  // before adding the new "changed" column.
+  // See: https://api.drupal.org/api/drupal/modules%21system%21system.install/function/system_update_136/5.x
+  if (db_index_exists('users', 'changed')) {
+    if (!db_index_exists('users', 'access')) {
+      db_add_index('users', 'access', array('access'));
+    }
+    db_drop_index('users', 'changed');
+  }
+
+  $spec = array(
+    'type' => 'int',
+    'not null' => TRUE,
+    'default' => 0,
+    'description' => 'Timestamp for when user was changed.',
+  );
+  $keys = array(
+    'indexes' => array(
+      'changed' => array('changed'),
+    ),
+  );
+
+  // In some cases sites have added the changed field themselves e.g. via a
+  // contrib or custom module. Ensure the field uses core's new schema.
+  if (db_field_exists('users', 'changed')) {
+    db_change_field('users', 'changed', 'changed', $spec, $keys);
+  }
+  else {
+    db_add_field('users', 'changed', $spec, $keys);
+    // Set the initial value for existing users.
+    db_update('users')
+      ->expression('changed', 'created')
+      ->execute();
+  }
+}
+
+/**
+ * Add a "changed" column to the user_admin View.
+ */
+function user_update_1025() {
+  // See if the content admin view has been modified on this site.
+  $config = config('views.view.user_admin');
+
+  // Only update if view isn't overridden (VIEWS_STORAGE_OVERRIDE).
+  if ($config->get('storage') == 2) {
+    return;
+  }
+
+  $data = array(
```

Comment:
Do we really need to add 100s of lines of code here and re-save the entire config file? Why can't we simply update only the things that need to be added/changed?

Reply:
Instead of the many lines of code, I suggest the following:

```php
$view = views_get_view('user_admin');
$view->revert();
```

This code will basically grab the current json file in the `views/config` and re-install it in the `files/config_XXX/active` folder.

Reply:
Yes, this is actually heaps better. We are only doing this when the view hasn't been modified, so reverting to the default view would work without breaking anything 👍🏼

Reply:
This is similar to an update in node.install which also includes the entire JSON in the update. I'm not sure if we want to assume that the Views module is loaded enough for that to work.